app/services/importers/create_cohort.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Importers
```

Comment:
I think we need to add `require "csv"` for the review app to work as it's falling over at the seeding stage

Reply:
Good catch tks @cwrw